bin/index.js

```diff
@@ -15,10 +15,18 @@ const { argv } = require('yargs')
       default: defaultBranch,
       description: `Use a different branch point other than ${defaultBranch}`,
     },
+    'enforce-all-errors': {
+      type: 'boolean',
+      default: false,
+      description: 'Enforce all commit message errors including normally ignored errors',
+    },
   });
 
 (async () => {
-  let formatted = await commitlint(argv);
+  let formatted = await commitlint({
```

Comment:
```
  let formatted = await commitlint({
    ...argv,
    shouldEnforceAllErrors: argv['enforce-all-errors'],
  });
  ```
  
  We get camel case for free, but since the new option is a bool, maybe we should prefix `should`?

Reply:
updated

Reply:
Looks like you forgot the `...argv,`

Reply:
it would require reassigning defaultBranch at function level - updated to your approach